cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/event/notifications/Notification.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2016 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ *
+ */
+
+package com.cloudant.sync.event.notifications;
+
+/**
+ * Marker interface for events.
+ */
+public interface Notification {
```

Comment:
I thought it made more sense to be `Notification` rather than event, because its in the `notifications` package.